1. Известно, что генеральная совокупность распределена нормально 
со средним квадратическим отклонением, равным 16. 
Найти доверительный интервал для оценки математического ожидания a с надежностью 0.95, 
если выборочная средняя M = 80, а объем выборки n = 256.

$\delta = \frac{t_{\lambda} \cdot \sigma}{\sqrt{n}}$

2Ф (t_lambda) = lambda_

In [18]:
M = 80

In [13]:
n = 256 

In [8]:
sigma_ = 16

In [9]:
lambda_ = 0.95

Ф (t_lambda) = 0.95 / 2

In [10]:
0.95/2

0.475

In [11]:
t_lambda = 1.96

In [16]:
delta = t_lambda * sigma_/(n**0.5)
delta

1.96

In [ ]:
Доверительный интервал:

In [19]:
M + delta

81.96

In [20]:
M - delta

78.04

ДИ = (78.04, 81.96)

In [24]:
import numpy as np

2. В результате 10 независимых измерений некоторой величины X, выполненных с одинаковой точностью, 
получены опытные данные:
6.9, 6.1, 6.2, 6.8, 7.5, 6.3, 6.4, 6.9, 6.7, 6.1
Предполагая, что результаты измерений подчинены нормальному закону распределения вероятностей, 
оценить истинное значение величины X при помощи доверительного интервала, покрывающего это 
значение с доверительной вероятностью 0,95.

In [25]:
a = np.array([6.9, 6.1, 6.2, 6.8, 7.5, 6.3, 6.4, 6.9, 6.7, 6.1])

In [26]:
a

array([6.9, 6.1, 6.2, 6.8, 7.5, 6.3, 6.4, 6.9, 6.7, 6.1])

In [28]:
a_mean = a.mean()
a_mean

6.590000000000001

In [29]:
a_std = (np.sum((a - a_mean)**2) / (10-1))**0.5
a_std

0.4508017549014448

In [30]:
a.std(ddof=1)

0.4508017549014448

In [32]:
t_kr = 2.262

$\delta = \frac{t_{\lambda} \cdot \sigma}{\sqrt{n}}$

In [34]:
delta_ = t_kr * a_std / (10 ** 0.5)
delta_

0.32246174408757394

In [35]:
a_mean - delta_

6.267538255912426

In [37]:
a_mean + delta_

6.912461744087575

ДИ = (6.267, 6.912)

3. Утверждается, что шарики для подшипников, изготовленные автоматическим станком, имеют средний диаметр 17 мм. 
Используя односторонний критерий с α=0,05, проверить эту гипотезу, если в выборке из n=100 шариков средний диаметр 
оказался равным 17.5 мм, а дисперсия известна и равна 4 мм.

4.Продавец утверждает, что средний вес пачки печенья составляет 200 г. 
Из партии извлечена выборка из 10 пачек. Вес каждой пачки составляет:
202, 203, 199, 197, 195, 201, 200, 204, 194, 190.
Известно, что их веса распределены нормально.
Верно ли утверждение продавца, если учитывать, что доверительная вероятность равна 99%?